curator/actions.py

```diff
@@ -581,7 +581,7 @@ def do_action(self):
         self.index_list.filter_forceMerged(
             max_num_segments=self.max_num_segments)
         self.index_list.empty_list_check()
-        self.loggit.info('forceMerging selected indices')
+        self.loggit.info('forceMerging {0} selected indices: {1}'.format(len(self.index_list.indices), self.index_list.indices)
```

Comment:
This line is not properly terminated, so the CI tests will not even run.

Reply:
that is to say, it needs an additional `)`

Reply:
Whoops, sorry. Edited on github 👎